operator/v1/types_ingress.go

```diff
@@ -246,6 +385,9 @@ type IngressControllerStatus struct {
 	// endpointPublishingStrategy is the actual strategy in use.
 	EndpointPublishingStrategy *EndpointPublishingStrategy `json:"endpointPublishingStrategy,omitempty"`
 
+	// securityProfileType is the actual security profile in use.
+	SecurityProfile *SecurityProfileType `json:"securityProfile,omitempty"`
```

Comment:
This is insufficient because it doesn't describe the totality of effective values used. See https://github.com/openshift/api/pull/322#issuecomment-500495748

Reply:
Understood. I intended on using `SecurityProfile *SecurityProfile` for status before the merge.